README.md

```diff
@@ -1,16 +1,14 @@
-Google App Engine Managed VMs Python Compat Runtime
+Google App Engine Flexible Environment Python Compat Runtime
 ===================================================
 
-Warning
--------
-The instructions here are for a multi-process version of the Python 2.7 runtime
-**currently in alpha**. To use the stable version of the runtime, follow the
```

Comment:
Until end-to-end tests are more extensive and cover at least essential AppEngine API functionality, I'm not comfortable calling this runtime Beta.